From e603dcbc1a545f6d6fb74d8d3cb5a66ca23b05ae Mon Sep 17 00:00:00 2001
From: brian greenfield <briang@cpan.org>
Date: Tue, 19 Jul 2016 15:57:51 +0100
Subject: [PATCH] Fix test 9 in basic.t with Git >= 2.9

[Git 2.9 Release Notes](https://github.com/git/git/blob/master/Documentation/RelNotes/2.9.0.txt)

> The output formats of "git log" that indents the commit log message by
> 4 spaces now expands HT in the log message by default. You can use
> the "--no-expand-tabs" option to disable this.

I've chosen the simple fix of matching the git message against
8 spaces or a TAB.


Bug: https://github.com/genehack/Git-Wrapper/issues/73
Origin: https://github.com/genehack/Git-Wrapper/pull/72
Bug-Debian: https://bugs.debian.org/834793


---
 t/basic.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/t/basic.t
+++ b/t/basic.t
@@ -79,7 +79,7 @@
 
 my $log = $log[0];
 is($log->id, (split /\s/, $rev_list[0])[0], 'id');
-is($log->message, "FIRST\n\n\tBODY\n", "message");
+like($log->message, qr/FIRST\n\n(?:        |\t)BODY\n/, "message");
 
 throws_ok { $git->log( "--format=%H" ) } q{Git::Wrapper::Exception};
 
